classes/spec_extensions.rb

```diff
@@ -85,6 +85,8 @@ def or_license_name_and_url
           return { license: "Apache 2", url: "https://www.apache.org/licenses/LICENSE-2.0.html" }
         elsif license.scan(/apache2/).count > 0
           return { license: "Apache 2", url: "https://www.apache.org/licenses/LICENSE-2.0.html" }
+        elsif license.scan(/apache 2/).count > 0
```

Comment:
@segiddins that has an extra required space,  you could do `apache (?:license, version )?2(?:\.0)` (added the "?:" for non-capturing), but then that won't match "apache2", since this still requires a space after apache. I would say splitting this into 2 would be the most readable:
- `/apache license, version 2(:?\.0)?/i`
- `/apache\s*2(:?\.0)?/i`
How about that?